### PR TITLE
Fix/cosmetic changes

### PR DESCRIPTION
**Pull request by arslancreo**

### CODE REVIEW

**Comment by KamathVishak96** [comment]
@arslancreo Please attach Screen Recordings/Screenshots of the changes made.

File: lib/ui/company page/company_page.dart [hunk 1]
```diff
@@ -472,59 +464,54 @@ class _CompanyPageState extends State<CompanyPage>
...
+                                ..onTap = () {
+                                  Navigator.push(
+                                      context,
+                                      MaterialPageRoute(
+                                        builder: (context) =>
+                                            const Subscription(),
+                                      ));
+                                },
+                              style: const TextStyle(
+                                  // color: bloc!.loadTheme == AppTheme.lightTheme
```
Comment (KamathVishak96) [inline lib/ui/company page/company_page.dart:507]: Get rid of these Line comments.

File: lib/ui/company page/company_page.dart [hunk 2]
```diff
@@ -591,7 +581,7 @@ class _CompanyPageState extends State<CompanyPage>
       builder: (context, snapshot) {
         if (snapshot.hasData) {
           bool isChartStyleTutShown = snapshot.data!
-              .getBool(PrefKeys.KEY_IS_CHART_STYLE_TUTORIAL_SHOWN) ??
+                  .getBool(PrefKeys.KEY_IS_CHART_STYLE_TUTORIAL_SHOWN) ??
```
Comment (KamathVishak96) [inline lib/ui/company page/company_page.dart:584]: Replace `!` with `?` and `?? false` with `== true`.

**Comment by arslancreo** [comment]
Screenshot 2022-10-31 at 4 08 05 PM

The cross button has been made equidistant from both the axes

**Review by KamathVishak96 — CHANGES_REQUESTED** [review]

File: lib/ui/company page/company_page.dart (hunk 1 above)
Comment (arslancreo) [reply lib/ui/company page/company_page.dart]: done

File: lib/ui/company page/company_page.dart (hunk 2 above)
Comment (arslancreo) [reply lib/ui/company page/company_page.dart]: done

**Review by KamathVishak96 — APPROVED** [review]

**Pull request merged**